### PR TITLE
feat(editor): limit when to show mapping tooltip

### PR DESCRIPTION
**Pull request by mutdmour**

### CODE REVIEW

File: packages/editor-ui/src/components/RunDataTable.vue [hunk 1]
```diff
@@ -199,9 +200,18 @@ export default mixins(externalHooks).extend({
 			draggedColumn: false,
 			draggingPath: null as null | string,
 			hoveringPath: null as null | string,
+			showMappingIntroHint: false,
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/RunDataTable.vue:203]: nitpick: Naming is a bit confusing here. I think `mappingHintVisible` would make more sense.

**Review by alexgrozav — DISMISSED** [review]
Working great! Nice work.

File: packages/editor-ui/src/components/RunDataTable.vue (hunk 1 above)
Comment (mutdmour) [reply packages/editor-ui/src/components/RunDataTable.vue:203]: makes sense.. ya I struggled a bit here with that. will update

**Review by alexgrozav — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.193.0`

**Pull request merged**